hppc/src/test/templates/com/carrotsearch/hppc/AbstractKTypeVTypeTest.java

```diff
@@ -0,0 +1,42 @@
+package com.carrotsearch.hppc;
+
+/**
+ * Unit helpers for <code>KType</code>, <code>VType</code> pair.
+ */
+/*! ${TemplateOptions.generatedAnnotation} !*/
+/*! #if ($TemplateOptions.KTypeGeneric) !*/
+@SuppressWarnings("unchecked")
+/*! #end !*/
+public class AbstractKTypeVTypeTest<KType, VType> extends AbstractKTypeTest<KType> {
```

Comment:
Contains existing methods that were copied between the KTypeVTypeXyzMapTest. Simple move, the code is the same.